tensorflow/contrib/layers/python/layers/layers.py

```diff
@@ -810,16 +866,14 @@ def bias_add(inputs,
                                        sc.original_name_scope, outputs)
 
 
-# TODO(jbms): change `rate` parameter to `dilation_rate` for consistency with
-# underlying op.
 @add_arg_scope
 def convolution(inputs,
                 num_outputs,
                 kernel_size,
                 stride=1,
                 padding='SAME',
                 data_format=None,
-                rate=1,
+                dilation_rate=1,
```

Comment:
This name change would not be backwards compatible. If changing the API, it should be done in a backwards compatible way (albeit maybe not in `contrib`... @martinwicke: what's the policy for breaking changes in contrib?).